openmmtools/testsystems.py

```diff
@@ -3473,6 +3474,82 @@ def __init__(self, constraints=app.HBonds, rigid_water=True, nonbondedCutoff=DEF
 
         self.system, self.positions = system, positions
 
+# =============================================================================================
+# Drew-Dickerson B-DNA dodecamer in explicit solvent
+# =============================================================================================
+
+class DNADodecamerExplicit(TestSystem):
+    """
+    Drew-Dickerson B-DNA dodecamer (CGCGAATTCGCG) in explicit solvent. Structure taken from the RCSB PDB accession code
+    4C64 (1.3 Angstrom resolution). Solvated using the TIP3P water model in a cuboidal box with at least a 10 Angstrom
+    clearance between the edge of the box and the DNA. The DNA is parametrized with the AMBER 14SB forcefield. The
+    system has a total charge of -22 as no neutralizing counterions have been included.
+
+    Parameters
+    ----------
+    constraints : optional, default=simtk.openmm.app.HBonds
+    rigid_water : bool, optional, default=True
+    nonbondedCutoff : Quantity, optional, default=DEFAULT_CUTOFF_DISTANCE
+    use_dispersion_correction : bool, optional, default=True
+        If True, the long-range disperson correction will be used.
+    nonbondedMethod : simtk.openmm.app nonbonded method, optional, default=app.PME
+       Sets the nonbonded method to use for the water box (one of app.CutoffPeriodic, app.Ewald, app.PME).
+    hydrogenMass : unit, optional, default=None
+        If set, will pass along a modified hydrogen mass for OpenMM to
+        use mass repartitioning.
+    switch_width : simtk.unit.Quantity with units compatible with angstroms, optional, default=DEFAULT_SWITCH_WIDTH
+        switching function is turned on at cutoff - switch_width
+        If None, no switch will be applied (e.g. hard cutoff).
+    ewaldErrorTolerance : float, optional, default=DEFAULT_EWALD_ERROR_TOLERANCE
+           The Ewald or PME tolerance.
+
+    Reference
+    ---------
+    Structure taken from Chem.Commun.(Camb.), 50, page 1794, 2014.
+    """
+
+    def __init__(self, constraints=app.HBonds, rigid_water=True, nonbondedCutoff=DEFAULT_CUTOFF_DISTANCE,
+                 use_dispersion_correction=True, nonbondedMethod=app.PME, hydrogenMass=None,
+                 switch_width=DEFAULT_SWITCH_WIDTH, ewaldErrorTolerance=DEFAULT_EWALD_ERROR_TOLERANCE, **kwargs):
+
+        TestSystem.__init__(self, **kwargs)
+
+        try:
+            from parmed.amber import AmberParm
```

Comment:
I copied the DHFR test system setup [here](https://github.com/choderalab/openmmtools/blob/master/openmmtools/testsystems.py#L3452-L3453). I realize this is not necessary, but I wanted to maintain consistency.